docs/limitations.rst

```diff
@@ -0,0 +1,27 @@
+Limitations
+------------------------------------
+
+Session Labeling
+~~~~~~~~~~~~~~~~
+Cloud Spanner Session Labeling is not supported.
+
+Request Priority
+~~~~~~~~~~~~~~~~
+Request priority can be set by unwrapping the Spanner-specific `SpannerConn` interface and setting the request priority as part of a db call.
+
+Tagging
+~~~~~~~
+Tags can be set by unwrapping the Spanner-specific `SpannerConn` interface and setting the tags using that interface.
+
+Partition Reads
+~~~~~~~
+Partition Reads can be done by unwrapping the Spanner-specific `SpannerConn` interface and doing the parition reads using that interface.
+
+PostgreSQL
+~~~~~~~
+Spanner databases that use the PostgreSQL dialect are not yet supported.
+
+Backups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Backups for databases are not managed by Go driver so it does not support it inherently.
+In general, none of the Spanner ORMs / drivers are expected to support backup management.
```

Comment:
Also here: Keep it simple. Users of `gorm` might not even know that there are other ORMs/drivers, and even if they do, they probably don't care.

```suggestion
Backups are not supported by this driver. Use the [Cloud Spanner Go client library](https://github.com/googleapis/google-cloud-go/tree/main/spanner) to manage backups programmatically.
```